docs/design.md

```diff
@@ -4,32 +4,34 @@ title: Connector Design Considerations
 
 The choices you make when creating a connector can include which superclass and dialect to use, and how you want to tune your connection using Tableau capabilities.
 
-## Choosing a Connection Class
+## Choose a connection class
 
-The class attribute is a unique key for your connector. When Tableau loads the connectors at startup, if the class has already been registered the connector will not be loaded. The class is also stamped in Tableau Workbook (.twb or .twbx) files and Tableau Datasource (.tds) files to identify what connector that particular connection was using.
+The <span style="font-family: courier new">class</span> attribute is a unique key for your connector. Make it something that is unlikely to be used by another connector. When Tableau loads the connectors at startup, if the class is already registered, the connector will not be loaded. The class is also stamped in Tableau Workbook (.twb or .twbx) files and Tableau Datasource (.tds) files to identify what connector that particular connection was using.
```

Comment:
This is an HTML tag, correct? I don't think this will work, these pages use [markdown](https://www.markdownguide.org/cheat-sheet/), not html.

Reply:
This was a style I saw elsewhere in the SDK, and repeated it throughout for consistency. (I understand that some html does work in MD docs.) If it doesn't render properly, I'll go back and replace this everywhere with a style that does work.